Gemfile

```diff
@@ -17,6 +17,8 @@ group :tester do
 
   # transports
   gem 'aws-sdk', '~> 2.2', '>= 2.2.12'
+  gem 'mime-types', '<= 2.99.1' # required by fog
```

Comment:
This should be `gem 'mime-types', '< 3'`. Really, `fog` should not be locking the old version, but if you *must* lock to a pre-3.0 release of mime-types, do so correctly in order to get the most up-to-date data and fixes on a regular basis.